.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Maybe add a comment above this line explaining what this does, and add a link to the relevant documentation.

Reply:
@waldyrious done.